resources/assets/js/components/JobBuilderImpact/JobBuilderImpact.tsx

```diff
@@ -0,0 +1,168 @@
+  },
+  healthCanada: {
+    id: "department.impactStatement.health",
+    defaultMessage:
+      "Health Canada is responsible for helping Canadians maintain and improve their health. It ensures that high-quality health services are accessible, and works to reduce health risks.",
+    description: "Department Impact Statement of Shared Services Canada",
+  },
+});
+
+const JobBuilderImpact: React.FunctionComponent<
+  JobBuilderImpactProps & InjectedIntlProps
+> = ({ intl, department, job }): React.ReactElement => {
+  return (
+    <section>
+      <ProgressTracker
+        items={items}
+        backgroundColor="black"
+        fontColor="white"
+        classNames="manager-jpb-tracker"
+        itemsWrapperClassNames="tracker manager-jpb-tracker-wrapper"
+      />
+      <div data-c-container="form" data-c-padding="top(triple) bottom(triple)">
+        <h3
+          data-c-font-size="h3"
+          data-c-font-weight="bold"
+          data-c-margin="bottom(double)"
+        >
+          <FormattedMessage
+            id="jobBuilder.impact.title"
+            defaultMessage="Create an Impact Statement"
+            description="Header of Job Poster Builder Impact Step"
+          />
+        </h3>
+        <ul data-c-margin="bottom(double)">
+          <li>
+            <FormattedMessage
+              id="jobBuilder.impact.points.opportunity"
+              defaultMessage="Working in the federal government offers an important opportunity to have a broad impact for Canadians."
+              description="Bullet Point on Job Poster Builder Impact Step"
+            />
+          </li>
+          <li>
+            <FormattedMessage
+              id="jobBuilder.impact.points.highlight"
+              defaultMessage="This is your chance to highlight what makes your work valuable and interesting."
+              description="Bullet Point on Job Poster Builder Impact Step"
+            />
+          </li>
+          <li>
+            <FormattedMessage
+              id="jobBuilder.impact.points.counts"
+              defaultMessage="Your impact statement is the first thing that applicants will see when they click your job poster so make sure it counts!"
+              description="Bullet Point on Job Poster Builder Impact Step"
+            />
+          </li>
+        </ul>
+        <p data-c-font-weight="bold" data-c-margin="bottom(normal)">
+          <FormattedMessage
+            id="jobBuilder.impact.header.department"
+            defaultMessage="How our department makes an impact:"
+            description="Header of Department Impact Section on Job Poster Builder Impact Step"
+          />
+        </p>
+        {/* <!-- The p tag below is where the dynamic department text goes (I used text from an old job poster as placeholder, but the real list of data is in the issue. --> */}
+        <p data-c-margin="bottom(double)">
+          {department !== undefined &&
+            intl.formatMessage(departmentImpactStatements[department])}
```

Comment:
Department displayed should be based on job.department_id.

Reply:
I know the current implementation allows for easily changing the department in storybook, but it doesn't make sense outside of storybook.

Reply:
The idea was to build the front end without linking it to jobs yet. The department impact statement should really come from the database, as the list of partner departments will change with every year and with every election.

Reply:
We're already passing in `job` and getting the impact statements from there. Consistency suggests we should get the department from there as well.